### PR TITLE
Update guides/contexts.md

### PR DESCRIPTION
**Pull request by kayuapi**

### CODE REVIEW

File: guides/contexts.md [hunk 1]
```diff
@@ -789,56 +793,58 @@ We defined a new `CartItemController` with the create and delete actions that we
...
-        where: c.user_uuid == ^user_uuid,
-        left_join: i in assoc(c, :items),
-        left_join: p in assoc(i, :product),
-        order_by: [asc: i.inserted_at],
-        preload: [items: {i, product: p}]
-      )
-    )
-  end
++ alias Hello.Catalog
+- alias Hello.ShoppingCart.Cart
```
Comment (kingshalaby1) [inline guides/contexts.md:797]: duplicate alias

Comment (kayuapi) [reply guides/contexts.md:797]: oh, that's actually a delete line

Comment (kingshalaby1) [reply guides/contexts.md:797]: ah, it's originally showing diff. I beg your pardon.

File: guides/contexts.md [hunk 2]
```diff
@@ -724,11 +726,13 @@ We won't focus on a real user authentication system at this point, but by the ti
 + alias Hello.ShoppingCart
 +
 + def fetch_current_cart(conn, _opts) do
-+   if cart = ShoppingCart.get_cart_by_user_uuid(conn.assigns.current_uuid) do
-+     assign(conn, :cart, cart)
-+   else
-+     {:ok, new_cart} = ShoppingCart.create_cart(conn.assigns.current_uuid)
-+     assign(conn, :cart, new_cart)
++   case ShoppingCart.get_cart_by_user_uuid(conn.assigns.current_uuid) do
```
Comment (chrismccord) [inline guides/contexts.md:729]: the `if` is preferred here :)

**Comment by chrismccord** [comment]
Closing as this has gone stale. Thanks!

**Pull request closed without merging**